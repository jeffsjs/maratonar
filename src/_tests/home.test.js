import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Home from '../pages/home';

describe('Home component', () => {
	it('Render Home', () => {
		const wrapper = shallow(<Home />);
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
