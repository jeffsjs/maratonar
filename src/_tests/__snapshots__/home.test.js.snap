// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component Render Home 1`] = `
<div
  className="home"
>
  <div
    className="header"
  >
    <h1
      className="row row-reverse"
    >
      <div
        className="logo"
      >
        <ForwardRef(SvgCalendarAlt) />
         Maratornar App
      </div>
    </h1>
  </div>
  <div
    className="row"
  >
    <h2>
      Seasons
    </h2>
  </div>
  <Connect(withRouter(ListSeries)) />
</div>
`;
